src/mpid/ch4/netmod/ofi/ofi_impl.h

```diff
@@ -302,17 +302,11 @@ MPL_STATIC_INLINE_PREFIX void MPIDI_OFI_win_datatype_unmap(MPIDI_OFI_win_datatyp
 
 MPL_STATIC_INLINE_PREFIX void MPIDI_OFI_win_request_complete(MPIDI_OFI_win_request_t * req)
 {
-    int count;
-    MPIR_Assert(HANDLE_GET_MPI_KIND(req->handle) == MPIR_REQUEST);
-    MPIR_Object_release_ref(req, &count);
-    MPIR_Assert(count >= 0);
-    if (count == 0) {
-        MPIDI_OFI_win_datatype_unmap(&req->noncontig->target_dt);
-        MPIDI_OFI_win_datatype_unmap(&req->noncontig->origin_dt);
-        MPIDI_OFI_win_datatype_unmap(&req->noncontig->result_dt);
-        MPL_free(req->noncontig);
-        MPIR_Request_free((req));
-    }
+    MPIDI_OFI_win_datatype_unmap(&req->noncontig->target_dt);
+    MPIDI_OFI_win_datatype_unmap(&req->noncontig->origin_dt);
+    MPIDI_OFI_win_datatype_unmap(&req->noncontig->result_dt);
+    MPL_free(req->noncontig);
+    MPIR_Request_free(req);
```

Comment:
Is it safe to do this? i.e. there's no cases where `req` has refcount more than one? I guess not, but just wanted to confirm.